packages/jellyfish-api-core/src/category/wallet.ts

```diff
@@ -62,6 +62,16 @@ export class Wallet {
     return await this.client.call('getbalance', ['*', minimumConfirmation, includeWatchOnly], 'bignumber')
   }
 
+  /**
+   * Returns an object with all balances
+   *
+   * @param {boolean} withTokens = false, Include tokens balances; Default is 'false' for backward compatibility.
```

Comment:
This means DFI exists in 2 forms, UTXO and Account form. 

You can ask @thedoublejay, @izzycsy, @canonbrother and @jingyi2811 what this means.